client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/customs-step/item-row.js

```diff
@@ -0,0 +1,111 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import TextField from 'woocommerce/woocommerce-services/components/text-field';
+import {
+	setCustomsItemDescription,
+	setCustomsItemTariffNumber,
+	setCustomsItemOriginCountry,
+} from 'woocommerce/woocommerce-services/state/shipping-label/actions';
+import {
+	getShippingLabel,
+	isLoaded,
+	getFormErrors,
+} from 'woocommerce/woocommerce-services/state/shipping-label/selectors';
+import CountryDropdown from 'woocommerce/woocommerce-services/components/country-dropdown';
+
+const TariffCodeHeader = localize( ( { translate } ) =>
+	<span>{ translate( 'Tariff Code' ) } (<a href="https://hts.usitc.gov/"
+		target="_blank" rel="noopener noreferrer">{ translate( 'look up' ) }</a>)</span>
+);
+
+const ItemRow = ( props ) => {
+	const {
+		errors,
+		packageId,
+		productId,
+		translate,
+		description,
+		defaultDescription,
+		tariffNumber,
+		originCountry,
+		countriesData,
+	} = props;
+
+	return <div className="customs-step__item-row">
+		<TextField
+			id={ packageId + '_' + productId + '_description' }
+			className="customs-step__item-description-column"
+			title={ translate( 'Description' ) }
+			value={ description }
+			placeholder={ defaultDescription }
+			updateValue={ props.setCustomsItemDescription }
+			error={ errors.description } />
+		<CountryDropdown
+			id={ packageId + '_' + productId + '_originCountry' }
+			className="customs-step__item-country-column"
+			title={ translate( 'Origin country' ) }
+			value={ originCountry }
+			updateValue={ props.setCustomsItemOriginCountry }
+			countriesData={ countriesData } />
+		<TextField
+			id={ packageId + '_' + productId + '_tariffNumber' }
+			className="customs-step__item-code-column"
+			title={ <TariffCodeHeader /> }
```

Comment:
How about calling `TariffCodeHeader` something like `TariffCodeTitle` instead to make it clearer, since we use it inside a `<TextField />` as the title.

Reply:
Changed in https://github.com/Automattic/wp-calypso/pull/24974/commits/e4f03d4415019f4ed65e7e7af8c8ed9394bf469f